src/assets/scss/components/_comment.scss

```
@@ -0,0 +1,63 @@
+.comment {
+    border: 1px solid #ccc;
+    padding: 40px;
+    margin-top: 20px;
+    margin-bottom: 10px;
+    border-radius: 5px;
+    box-shadow: 0 0 5px rgba(0, 0, 0, 0.1);
+
+    &-header {
+        display: flex;
+        align-items: center;
+        margin-bottom: 1px;
+
+        .avatar-content {
+            width: 10px;
+            height: 10px;
+            border-radius: 50%;
+            margin-right: 10px;
+        }
+
+        .comment-meta {
+            flex-grow: 1;
+        }
+    }
+
+    &-body {
+        margin-left: 30px;
+    }
+
+    &-time {
+        color:#6a9cd2;
+        margin-bottom: 10px;
+    }
+
+
+    &-actions {
+
+        .btn {
+            padding: 5px 10px;  
+            border: none;
+            cursor: pointer;
+            border-radius: 3px;
+        }
+
+        .btn-view {
+            background-color: #007bff;
+            color: #fff;
+        }
+
+        .btn-edit {
+            background-color: #ffc107;
+            color: #000;
+        }
+
```

If you want to add a dark theme style, you can just simply add `html[data-bs-theme="dark"]`.
For example:

```suggestion
// Dark theme
html[data-bs-theme="dark"] .comment {
  border: 1px solid #404053;
}
```